libs/rttrConfig/src/RTTR_Version.cpp

```diff
@@ -5,20 +5,41 @@
 #include "RTTR_Version.h"
 #include "build_version_defines.h"
 
-namespace rttr { namespace version {
-
-    std::string GetTitle() { return "Return To The Roots"; }
```

Comment:
Shame it starts splitting those functions although I'm not sure why it didn't [do that before](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#allowshortfunctionsonasingleline):

> SFS_Inline (in configuration: Inline) Only merge functions defined inside a class. Implies “empty”.

Reply:
Either set that to "All" (which changes a lot) or use `// clang-format off`. Those seem to be the only options.

Reply:
should be fine